src/data_collection/create_map.py

```diff
@@ -0,0 +1,149 @@
+import ee
+import geemap.foliumap as geemap
+from geopy.geocoders import Nominatim
+from geopy.exc import GeopyError
+from decouple import config
+from constants import Cloud_Coverage, Spatial_Res, Geo_Tolerance, temperature_palette, temp_ranges
+
+HEAT_MAP_GEE_PROJECT = config("GEE_PROJECT")
+ee.Initialize(project=HEAT_MAP_GEE_PROJECT)
+
+
+def get_city_coordinates(city: str) -> list[int]:
```

Comment:
`float`? this type hint is too flex, this allows for a list of >2 ints, better to make it more strict + readable using named tuples:

Python namedtuple with typing #Shorts
https://www.youtube.com/watch?v=ljQHjQx76Po

Use namedtuples for more readable code
https://www.youtube.com/watch?v=dsGwfjZxp9A